rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -119,11 +132,49 @@ def get_all_info(
             entries_limit=FREE_NFT_LIMIT,
         )
 
+    def get_db_nft_balances(self, filter_query: NFTFilterQuery) -> Dict[str, Any]:
+        """Filters (with `filter_query`) and returns cached nft balances in the nfts table"""
+        entries = defaultdict(list)
+        query, bindings = filter_query.prepare()
+        with self.db.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT identifier, name, last_price, last_price_asset, manual_price, '
+                'owner_address, is_lp, image_url, collection_name FROM nfts ' + query,
+                bindings,
+            )
+            for db_entry in cursor:
+                price_in_asset = FVal(db_entry[2])
+                # Asset should always exist since it is guaranteed by the db schema
+                price_asset = Asset(db_entry[3])
+                # find_usd_price should be fast here since in most cases price should be cached
+                usd_price = price_in_asset * Inquirer.find_usd_price(price_asset)
```

Comment:
So for usd price you removed it from saving in the DB and instead you query it for each nft when returning the balances?